src/mscorlib/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -592,11 +592,19 @@ private void Resize(int newSize, bool forceNewHashCodes)
 
         public bool Remove(TKey key)
         {
+            TValue unused;
+            return Remove(key, out unused);
```

Comment:
The actual C# 7.0 syntax is `out var _`. Using that seems to compile fine for me.